### PR TITLE
Create day-0-maxine.py script

### PR DESCRIPTION
**Pull request by maxin-e**
Cool workshop! Thanks :)

### CODE REVIEW

File: day-0/day-0-maxine.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+def draw_multicolour_square(t, sz):
+    for x in range(0, 4):
+        t.forward(sz)
+        t.left(90)
+
+
+wn = turtle.Screen()
+
+tess = turtle.Turtle()
+tess.pensize(7)
```
Comment (BogDAAAMN) [inline day-0/day-0-maxine.py:13]: Hey Maxine, maybe 5????

File: day-0/day-0-maxine.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import turtle
+
+
+def draw_multicolour_square(t, sz):
+    for x in range(0, 4):
+        t.forward(sz)
+        t.left(90)
+
```
Comment (gijsvd) [inline day-0/day-0-maxine.py:8]: Interesting

File: day-0/day-0-maxine.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+size = 32
+for i in range(20):
+    draw_multicolour_square(tess, size)
+    size = size + 20
+    tess.forward(25)
+    tess.right(18)
+
+turtle.mainloop()
+
```
Comment (gijsvd) [inline day-0/day-0-maxine.py:24]: Amazing

File: day-0/day-0-maxine.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+tess = turtle.Turtle()
+tess.pensize(7)
+tess.left(90)
+
+size = 32
+for i in range(20):
+    draw_multicolour_square(tess, size)
+    size = size + 20
+    tess.forward(25)
+    tess.right(18)
```
Comment (BogDAAAMN) [inline day-0/day-0-maxine.py:21]: noice loop

**Comment by BogDAAAMN** [comment]
can we work copy a little bit

**Pull request merged**